packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -124,6 +129,19 @@ export class VaButton {
     return this.text;
   };
 
+  private handleSubmit() {
+    const theForm = this.el.closest('form');
+    const submitEvent = new CustomEvent('submit', {
+      bubbles: true,
+      cancelable: true,
+      composed: true,
+    });
+    theForm.dispatchEvent(submitEvent);
+    if (!this.dontSendForm) {
```

Comment:
Can you explain here why this is necessary so that we can document the team or form's library needs?